### PR TITLE
Minor copy edits and removal of tagline

### PR DESCRIPTION
**Pull request by mrniaboc**

### CODE REVIEW

File: app/pages/home.cjsx [hunk 1]
```diff
@@ -16,34 +16,24 @@ counterpart.registerTranslations 'en',
...
+        content:'''Professional researchers and volunteers share our [discussion boards](./#/talk), using them to explore and analyse project data. Much of the most exciting research produced by the Zooniverse originates from these partnerships.'''
       discover:
         title: 'Discover, teach, and learn'
         content: '''Our platform offers many opportunities for education, from using projects
-        in classrooms to sharing information between volunteers. You can even use the Project
-        Builder to create your very own Zooniverse project!'''
+        in classrooms to sharing information between volunteers. You can even use the [Zooniverse Project Builder](./#/lab) to create your very own project!'''
     featuredProjects:
       title: 'Get started on a project right now!'
-      tagline: 'These are just a few of our projects.'
```
Comment (srallen) [inline app/pages/home.cjsx]: Need to remove tagline from the render too.

Comment (mrniaboc) [inline app/pages/home.cjsx]: @srallen I'm not sure what that means. Can you point me in the right direction?

Comment (edpaget) [inline app/pages/home.cjsx]: Delete [this line](https://github.com/zooniverse/Panoptes-Front-End/blob/mrniaboc-copy-edits/app/pages/home.cjsx#L75)

Comment (mrniaboc) [inline app/pages/home.cjsx]: Thanks @edpaget

**Comment by edpaget** [comment]
You'll also need to rebase this branch in git.

**Comment by mrniaboc** [comment]
Oh dear...

**Comment by mrniaboc** [comment]
Rebased @edpaget

**Pull request merged**